js/arena.js

```diff
@@ -0,0 +1,63 @@
+console.log('its work');
+import gladiator from './gladiator.js';
+export default class arena extends gladiator {
+
+constructor(name ,gladiators = []){
+    super();
+    this.name = name.charAt(0).toUpperCase() + name.slice(1);
+    this.gladiators = gladiators;
+}
+
+addGladiator(glad){
+    if (this.gladiators.length <= 1) {
+        this.gladiators.push(glad);
+    }
+}
+removeGladiators(glad){
+    this.gladiators.pop(glad);
```

Comment:
pop() will remove the last element, and regardless of the argument. You can use splice or shift.